### PR TITLE
Luby sequence cooldowns

### PR DESCRIPTION
**Pull request by lixitrixi**
Add a luby sequence generator and use it to decide API polling cooldown times.

### CODE REVIEW

File: static/client.js [hunk 1]
```diff
@@ -69,15 +71,36 @@ class ConjureClient {
             let poll = () => {
                 this._poll(jobid)
                     .then(res => {
-                        if (res.status == "wait") {
-                            setTimeout(poll, 2000); // Poll every second
+                        if (true) { // res.status == "wait"
```
Comment (ozgurakgun) [inline static/client.js:74]: why true here instead of == "wait" as before?

**Review by ozgurakgun — COMMENTED** [review]
I like it! One minor comment/question.

File: static/client.js (hunk 1 above)
Comment (lixitrixi) [reply static/client.js:74]: whOOPs that was a test to keep it polling so I could see the sequence in action!

File: static/client.js [hunk 2]
```diff
@@ -70,14 +72,35 @@ class ConjureClient {
                 this._poll(jobid)
                     .then(res => {
                         if (res.status == "wait") {
-                            setTimeout(poll, 2000); // Poll every second
+                            console.log("wait")
+                            setTimeout(poll, _LUBY_GEN.next().value * _LUBY_MULT); // Poll every second
```
Comment (ozgurakgun) [inline static/client.js:76]: also this shouldn't say every second any more

**Comment by lixitrixi** [comment]
That's now fixed! Thank you for noticing these haha I must've been way too tired to notice them.

**Pull request merged**